### PR TITLE
Backport Examples and Install files for 2.8

### PR DESCRIPTION
**Pull request by OwenCorrigan76**
Backport examples and install files for 2.8 from Upstream Strimzi to Streams for Apache Kafka

### CODE REVIEW

File: install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml [hunk 1]
```diff
@@ -55,37 +57,37 @@ spec:
...
+              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0
             - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
-              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0
+              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0
             - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
-              value: registry.redhat.io/amq-streams/bridge-rhel9:2.7.0
+              value: registry.redhat.io/amq-streams/bridge-rhel9:2.8.0
             - name: STRIMZI_DEFAULT_MAVEN_BUILDER
-              value: registry.redhat.io/amq-streams/maven-builder-rhel9:2.7.0
+              value: registry.redhat.io/ubi8/openjdk-17:1.16
```
Comment (ppatierno) [inline install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:90]: keep amq-streams stuff

**Review by ppatierno — CHANGES_REQUESTED** [review]

File: install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml (hunk 1 above)
Comment (ppatierno) [reply install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:90]: but we should have a 2.8.0 image for that, or?

Comment (OwenCorrigan76) [reply install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml]: Yes, changed now

**Review by ppatierno — APPROVED** [review]

**Review by kyguy — APPROVED** [review]
Would it be worth copying the variable expansion done in the env var section of the OLM deployment files for the install deployment file?

**Comment by OwenCorrigan76** [comment]
@kyguy Like this:https://github.com/jboss-container-images/amqstreams-1-openshift-image/blob/0967b930ce7016ca8b6d835c7dd073238015ba76/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml#L125C13-L129C41

**Pull request merged**